cmd/juju/application/series_selector.go

```diff
@@ -116,3 +143,25 @@ func (s seriesSelector) userRequested(requestedSeries string) (string, error) {
 	logger.Infof(msgUserRequestedSeries, series)
 	return series, nil
 }
+
+func (s seriesSelector) validateSeries(seriesName string) error {
+	// if we're forcing then we don't need the following validation checks.
+	if len(s.supportedJujuSeries) == 0 {
+		// programming error
+		return errors.Errorf("expected supported juju series to exist")
+	}
+	if s.force {
+		return nil
+	}
+
+	var found bool
+	for _, supportedSeries := range s.supportedJujuSeries {
+		if seriesName == supportedSeries {
+			found = true
```

Comment:
I think I mentioned this on the original PR as well; you could add a break here

Reply:
I'm sure I put that in, I'll add it to this and a port forward will get it